File: typescript-angular/API/Client/CloudQuery.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface CloudQuery {

        "key"?: string;

        "skip"?: number;

        "limit"?: number;

        "query"?: Query;

        "sort"?: any;

        "select"?: any;
    }

}

File: akka-scala/src/main/scala/io/swagger/client/model/CloudQuery.scala
package io.swagger.client.model

import io.swagger.client.core.ApiModel
import org.joda.time.DateTime


case class CloudQuery (
  key: Option[String],
  skip: Option[Int],
  limit: Option[Int],
  query: Option[Query],
  sort: Option[Any],
  select: Option[Any])
   extends ApiModel


